src/api/app/controllers/cloud/upload_jobs_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Cloud
+  class UploadJobsController < ApplicationController
+    before_action :require_login
+    before_action -> { feature_active?(:cloud_upload) }
+    before_action :validate_configuration_presence, only: [:index, :create]
```

Comment:
The only is not necessary as there are only the :index and :create actions, right?

Reply:
Oh, right